app/openlxp_xss_project/settings.py

```diff
@@ -0,0 +1,166 @@
+"""
+Django settings for openlxp_xss_project project.
+
+Generated by 'django-admin startproject' using Django 3.2.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+import mimetypes
+import os
+import sys
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
Just checking -- Is this double parent reference correct?

Reply:
Yup, this is boilerplate code from the Django install